src/components/UserProfile/UserLinks/UserLinks.jsx

```diff
@@ -1,30 +1,35 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
-const UserLinks = ({
-  links = [],
-}) => (
+const UserLinks = ({ links = [] }) => (
   <>
     <div className="linkContainer">
       {links.map((item, index) => {
         if (item.Link.includes('http')) {
-          return (
-            <React.Fragment key={item.Name}>
-              <a key={item.link} href={item.Link} target="_blank" rel="noreferrer">
-                {item.Name.toUpperCase()}
-              </a>
-              <br />
-            </React.Fragment>
-          );
+          // Render external link if it's not an empty string
+          if (item.Link.trim() !== '') {
+            return (
+              <React.Fragment key={item.Name}>
+                <a key={item.link} href={item.Link} target="_blank" rel="noreferrer">
+                  {item.Name.toUpperCase()}
+                </a>
+                <br />
+              </React.Fragment>
+            );
+          }
+        } else {
+          // Check if the link is an internal link and not an empty string
```

Comment:
During the test I found if an added link doesn't start with "http", it redirects to internal page. But the link I've added is `www.google.com`. I'm just wondering if that's related to this PR?

Reply:
It's not related to this PR(I think this part of code was modified before, so i updated it in this way to resolve the conflicts). In UserLinks.jsx, it's set that the links need to include 'http', otherwise it will redirect to internal page.